src/client/FlagsClient.ts

```diff
@@ -20,6 +21,7 @@ class FlagsClient {
 
   constructor(public config: FlagsConfig) {
     this.checkValidInstance();
+    this.config.uri = config.uri || '/client/features';
```

Comment:
Same as https://github.com/fvoordeckers/react-unleash-flags/pull/14/files#r615737514 
I think a constant should be great here